draft-ietf-rtcweb-jsep.xml

```diff
@@ -2186,8 +2200,7 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
 
           <section title="VoiceActivityDetection" anchor="sec.voiceactivitydetection2">
             <t>Silence suppression in the answer is handled as described in
-              <xref target="sec.voiceactivitydetection1"></xref>.
-            </t>
+            <xref target="sec.voiceactivitydetection1"></xref>.</t>
```

Comment:
Since you can't answer with CN if it wasn't offered, we probably need a note here to the effect of:
"If silence suppression was not indicated in the offer, via the omission of CN codecs or relevant fmtp parameters, the VoiceActivityDetection parameter has no effect, and the answer should be generated as if VoiceActivityDetection was set to false."